ExpressMapper.Tests NET40/CollectionTests.cs

```diff
@@ -31,6 +31,64 @@ public void AutoMemberMap()
         }
 
         [Test]
+        public void AutoMemberMap_ToNonGenericListInherited() {
+          Mapper.Register<TestModel, TestViewModel>();
+          Mapper.Register<Size, SizeViewModel>();
+          Mapper.Register<Country, CountryViewModel>();
+          Mapper.Compile();
+
+          var testData = Functional.CollectionAutoMemberMap();
+
+          var result = Mapper.Map<List<TestModel>, NonGenericCollectionInhertedFromList>( testData.Key );

```

Comment:
As mentioned earlier, that was not the scenario i was trying to solve. But i will *add* a new test for those cases as well.